Translations/pl.lproj/advanced.strings

```diff
@@ -15,7 +15,7 @@
 "2270.title" = "Podziel plik:";
 
 /* Class = "NSComboBoxCell"; placeholderString = "Example: 5 MB"; ObjectID = "2271"; */
-"2271.placeholderString" = "np. 5 MB";
+"2271.placeholderString" = "max. rozmiar 1 pliku - np. 5 MB";
```

Comment:
@lboratynski What's the idea behind this addition?

Reply:
In my opinion, such a notation is more clear and makes it obvious that the file will be divided into several parts. This is just a suggestion, the previous notation is also fine.

Reply:
For better consistency with the rest of the translations I prefer to leave it as "np. 5 MB"

Reply:
Sure, as you wish. That's actually the best option.